transports/quic/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "libp2p-quic"
+version = "0.7.0"
+authors = ["David Craven <david@craven.ch>", "Parity Technologies <admin@parity.io>"]
```

Comment:
Yes, that's right.

Reply:
Okay. To avoid confusion in the future, e.g. in case of questions concerning the code, I am removing David as an author here. 
@dvc94ch please let me know in case that I missed parts in the code that are still authored by you.